scripts/synapse_port_db

```diff
@@ -959,7 +959,7 @@ class Porter(object):
         def r(txn):
             txn.execute(
                 "ALTER SEQUENCE event_auth_chain_id RESTART WITH %s",
-                (curr_chain_id,),
+                (curr_chain_id + 1,),
```

Comment:
this was added in #9449, but I think it was wrong. compare with other `ALTER SEQUENCE` incantations in this script.